src/cmd/run.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/spf13/cobra"
+
+	"github.com/defenseunicorns/uds-cli/src/pkg/runner"
+	"github.com/defenseunicorns/uds-cli/src/types"
+)
+
+var (
+	// todo: add to config
+	actionsFileLocation string
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run [ TASK NAME ]",
+	Short: "run a task",
+	Long:  `run a task from an actions file`,
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var actionsFile types.ActionsFile
+
+		err := utils.ReadYaml(actionsFileLocation, &actionsFile)
+		if err != nil {
+			panic(err)
```

Comment:
I used panic for speed with wren, it should be changed to follow the pattern of uds-cli. Only putting the comment here but there are many places

Reply:
![Screenshot 2023-10-31 at 6 29 11 PM](https://github.com/defenseunicorns/uds-cli/assets/42304551/159a0a28-1734-4c59-b0c3-9ec9e3794850)
Good catch! I wasn't paying attention and my IDE automatically hid those error checks

Reply:
fixed